src/styles/blocks/header.scss

```diff
@@ -0,0 +1,56 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
```

Comment:
I understand that the header and logo are small BEM blocks
But according to BEM rules, each BEM block should be in a separate file

So they should be in separate files (or make the logo just an element of the header block)

Not critical and everything else looks good so approved ✔️ 
But remember this in the future 🙂